docs/hstaxe/installing.rst

```diff
@@ -84,13 +84,13 @@ Note that you may also need to install the
 [c-blosc](https://github.com/Blosc/c-blosc) package as a dependency
 of the [tables](https://pypi.org/project/tables/) package on certain
 platforms (e.g., this is currently needed for macOS and Windows with
-Python 3.9 due to missing `tables` wheels on PyPI).
+Python 3.9 due to missing ``tables`` wheels on PyPI).
```

Comment:
Is this statement (paragraph) still correct? Especially for MacOS? We have seen successful builds with Python 3.9 now, right?

Reply:
No idea; just copied it over from the README 😉

Reply:
I don't believe it's true for Mac, could you check for Windows and delete this if it's no longer needed @duytnguyendtn ?

Reply:
Good question 😅

Reply:
Yes, good question indeed! I do not have access to any windows machine unfortunately!

Reply:
@pllim reminded us offline that she had tried to install on Windows, and it failed because the `wcstools` available on conda-forge doesn't have a Windows build. So I think we add a caveat to the docs that the current version of `hstaxe` isn't Windows compatible and Windows users should use one of the legacy install options instead.

Reply:
Implemented in https://github.com/spacetelescope/hstaxe/pull/50/commits/ddc24d4e79e85406bd3a15189b38da8d8cf9ce31